### DISCUSSION TITLE
de/2-01-kata-0/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#53] [General]
# de/2-01-kata-0/

https://ai.lernos.org/de/2-01-kata-0/

**Comment by ghost** [comment]
Typo in Punkt 3: Frequent → Frequenz

**Comment by ghost** [comment]
Bei den ersten drei Aufzählungspunkten für Woche 0 fehlt am Ende das Satzendezeichen (.).

**Comment by Flesch-04-18** [comment]
Hallo, vielleicht wäre hier ein kleiner Abschnit sinnvoll was es mit den Grundlagen auf sich hat und wie sie durch die Kata führen werden.

In einem anderen Kata hatte ich schon gesehen das gefragt wird ob die Aufgaben nummeriert werden könnten.
Das könnte man für die Grundlagen auch machen und sie Clustern oder in eigene Bereiche verschieben:

01

KI & Gesellschaft

02

KI Modelle

KI-Tools und Services,

03

KI & Maschinelles Lernen

...

Hintergrund: ich habe eben erst in Kata 2 bemerkt das sich die Katas dann auf spezielle Texte aus den Grundlagen beziehen.
Davor hatte ich mich aber schon einmal mit den Grundlagen beschäftigt so wie hier unter "So kannst Du mitmachen" >> "Schritt 5" beschrieben wurde.
https://loscon.lernos.org/de/kimooc24/
Zusätzlich habe ich mir die Grundlagen intuitiv von oben nach unten angeschaut und das ist ein heftiger Einstieg für Neulinge in dem Thema.
Ki & Maschinelles Lernen > Neuronale Netze > Maschinelles vs. Menschliches Lernen ...

**Comment by SylwiaMiedza** [comment]
Im folgenden Satz fehlt ein Komma [,]:

"Die wichtigste Aufgabe in der ersten Woche ist es [,] Euch gegenseitig besser kennenzulernen."
